packages/spaces/commands/vpn/info.js

```diff
@@ -25,45 +25,46 @@ function displayVPNInfo (space, info) {
     return colored
   }
 
-  cli.styledHeader(`${space} VPN Info`)
+  cli.styledHeader(`${name} VPN Info`)
   cli.styledObject({
     ID: info.id,
     'Public IP': info.public_ip,
     'Routable CIDRs': format.CIDR(info.routable_cidrs),
-    State: `${sF(info.state)}`,
-    'Provisioning Status': info.status,
+    'Status': info.status,
```

Comment:
Thoughts on s/Provisioning Status/Status?

Reply:
Makes sense to me! 👍